templates/boot-be/generic-uboot-tftp-ramdisk-boot-be-template.jinja2

```diff
@@ -16,7 +16,5 @@
     method: u-boot
     commands: ramdisk
     prompts:
-      - 'linaro-test'
```

Comment:
So we don't need "linaro-test" anywhere anymore?

Reply:
Never did.